common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -91,7 +95,34 @@ def fetch_file_if_present(filename, fetch_submodules: false)
         raise Dependabot::DependencyFileNotFound, path
       end
 
+      def load_cloned_file_if_present(filename)
+        path = File.join(clone_repo_contents, filename)
```

Comment:
Might also need to join in the directory here if it's set, something like:

```suggestion
        path = File.join(clone_repo_contents, directory, filename)
```